pkg/controller/plan/kubevirt.go

```diff
@@ -1584,6 +1584,15 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	if err != nil {
 		return
 	}
+	if vm.VM.MultiBootDiskNumber != 0 {
```

Comment:
I suppose this should change, we need to match the selected disk name to the number it will be assigned